### ISSUE TITLE
Not able to quit Syncthing Android when no Wi-Fi

### ISSUE THREAD

**Issue by reportxx** [syncthing/syncthing-android#1814]
Steps to reproduce:

1. Disconnect Wi-Fi
2. Open Syncthing Android
3. Syncthing disabled message is shown because there is no Wi-Fi
4. Clicking Exit
5. Syncthing window closes but Syncthing Android is still running and is not possible to quit.

I must connect to Wi-Fi to quit Syncthing Android.

**Comment by fps** [comment]
How do you tell it's still running? Because it is in the list of recent tasks?

**Comment by reportxx** [comment]
Becaise a persistent "Syncthing running" notification appears in the notification area. See screenshot
![syncthing running](https://user-images.githubusercontent.com/84130654/184672891-ccf3e9d0-e117-48ce-9774-0c1d09568a77.jpg)
.

**Comment by fps** [comment]
Interesting. In this case it should really say "Syncthing disabled". Is this maybe a translation error?

**Comment by reportxx** [comment]
Sorry, it already says "Syncthing disabled" so the last comment and screenshot is wrong. The only problem is that there is no way to close the application when the Syncthing disabled message appears because there is no Wi-Fi or mobile data.

**Comment by fps** [comment]
Yeah, that botheres me, too. I guess it would be nice if the "Exit" button on the dialog would really exit the app. Maybe making the dialog have three buttons instead of two worded like "Change settings", "Leave running" and "Exit" would be more appropriate. What do the maintainers think?

**Comment by imsodin** [comment]
Yup, quit or cancel for what's exit now seems more appropriate. And in case the app is not configured to run on boot/as service, then an exit button that actually does the same thing as the menu exit button seems sensible too.

**Comment by gitgt** [comment]
Still a problem in v 1.23.4.

**Comment by fps** [comment]
Hi,

those were pushes to my fork IIRC. Those changes did not make it into syncthing.

**Comment by reportxx** [comment]
When will it be pushed into syncthing?
